docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
-## Quickstart
 
-Hi! Glad to see you made it to the Quickstart guide! In this guide you are introduced to how CARTOframes can be used by data scientists in spatial analysis workflows. Using bike share data, this guide walks through some common steps a data scientist takes to answer the following question: are the company's bike share stations placed in optimal locations?
+# Quickstart
 
-Before you get started, we encourage you to have your environment ready so you can get a feel for the library by using it. If you don’t have your environment set-up yet, check out this guide first. You will need:
+## Introduction
+Hi! Glad to see you made it to the Quickstart guide! In this guide you are introduced to how CARTOframes can be used by data scientists in spatial analysis workflows. Using fake Starbucks revenue data, this guide walks through some common steps a data scientist takes to answer the following question: which stores are performing better than others?
 
-- A python notebook environment
-- The CARTOframes library installed
+Before you get started, we encourage you to have CARTOframes installed so you can get a feel for the library by using it:
+
+```
+pip install --pre cartoframes
+```
+
+If you don’t have your python environment set-up yet, check out this guide first.
 
 ### Spatial analysis scenario
 
-Let's say you work for a bike share company in Arlington, Virginia and you want to better understand how your stations around the city are being used, and if these stations are placed in optimal locations.
+Let's say you are a data scientist working for Starbucks and you want to better understand why some stores at Brooklyn, New York, perform better than others.
 
 To begin, let's outline a workflow: 
 
-- Explore your company's data
-- Discover and enrich data thanks to the CARTO catalog
-- Analyse if the current bike stations are placed in optimal locations
+- Get and explore your company's data
+- Create the areas of influence for your stores
```

Comment:
since you mention Data Observatory in the next bullet, here do you want to mention something like:
"Create areas of influence for your stores using CARTO's data services" or whatever you think appropriate

Reply:
True, I have remove CARTO.
- Create areas of influence for your stores
- Enrich your data with population data